src/components/layout/OrgHeader.tsx

```diff
@@ -66,7 +65,8 @@ const OrgHeader = ({ following, onFollow, onUnfollow, org } : OrgHeaderProps) :
                                 </Button>
                             ) }
                         </>
-                    ) : <SignupDialogTrigger />  }
+                    //TODO: Create button alternative for non-users
+                    ) : <></>  }
```

Comment:
I'm not used to seeing this approach to rendering an empty element. I think you should just use `null` instead.